### PR TITLE
Import emcpy modules

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
## Description

Replaced all plotting dependency codes within `eva` to be imported by `emcpy`.

This will close issue #57

### CODE REVIEW

File: src/eva/diagnostics/density.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from eva.eva_path import return_eva_path
 from eva.utilities.config import get
 from eva.utilities.utils import get_schema, update_object, slice_var_from_str
-import eva.plot_tools.plots
+from emcpy.plots.plots import Density as DensityPlot
```
Comment (CoryMartin-NOAA) [inline src/eva/diagnostics/density.py]: Is there a reason why not just keeping it as `emcpy.plots.plots.Density`? Or is that too verbose?

Comment (kevindougherty-noaa) [reply src/eva/diagnostics/density.py]: I tried that, and it fails. I get the following error:
`ModuleNotFoundError: No module named 'emcpy.plots.plots.Scatter'; 'emcpy.plots.plots' is not a package`

Comment (CoryMartin-NOAA) [reply src/eva/diagnostics/density.py]: No wouldn't it be `import emcpy.plots.plots`? and then `emcpy.plots.plots.Scatter()`? If that doesn't work, what you have is obviously fine. Was just curious.

Comment (kevindougherty-noaa) [reply src/eva/diagnostics/density.py]: I'll try that.

Comment (CoryMartin-NOAA) [reply src/eva/diagnostics/density.py]: Thanks, I think that's clearer for the developers where the method comes from

**Review by CoryMartin-NOAA — APPROVED** [review]
Looks good to me, and all tests seem to pass. Will want @danholdaway to confirm things look good before merging in.

**Review by danholdaway — APPROVED** [review]

**Pull request merged**